trackman/templates/auth/view_sessions.html

```diff
@@ -0,0 +1,36 @@
+{% set page_title="View Sessions" %}
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container">
+    <section>
+        <header>
+            <h2>View Sessions</h2>
+        </header>
+
+        <p>This is a list of devices that have logged into your account. Revoke
+        any session that you do not recognize.</p>
+
+        <div class="list-group">
+{% for sess in sessions %}
+            <div class="list-group-item">
+                <div class="d-flex w-100 justify-content-between">
+                    <h5 class="mb-1">{{ sess.remote_addr }}</h5>
+                    {% if sess.token == current_session_token %}
+                    <small>Your current session</small>
+                    {% else %}
+                    <form action="{{ url_for('.revoke_session', session_id=sess.id) }}" method="post" class="form-inline">
```

Comment:
any reason we're not using wtforms here? just too trivial?

Reply:
Too trivial, yes.